xbmc/addons/kodi-dev-kit/include/kodi/versions.h

```diff
@@ -109,7 +109,7 @@
                                                       "addon-instance/ImageDecoder.h"
 
 #define ADDON_INSTANCE_VERSION_INPUTSTREAM            "3.1.0"
-#define ADDON_INSTANCE_VERSION_INPUTSTREAM_MIN        "3.1.0"
+#define ADDON_INSTANCE_VERSION_INPUTSTREAM_MIN        "3.1.1"
```

Comment:
If you increase ADDON_INSTANCE_VERSION_INPUTSTREAM_MIN , you must also increase ADDON_INSTANCE_VERSION_INPUTSTREAM.

But more important, the bump for the PVR API version is still completely missing.

Reply:
Yes, that was a mistake, apologies. I also misunderstood if you just wanted the bump on InputStream or back on both. Both will be corrected.